src/ploomber/entrypoint.py

```diff
@@ -62,13 +62,17 @@ def find_entry_point_type(entry_point):
     if type_:
         return type_
     else:
-        raise ValueError(
-            'Could not determine the entry point type from value: '
-            f'{entry_point!r}. Expected '
-            'an existing file with extension .yaml or .yml, '
-            'existing directory, glob-like pattern '
-            '(i.e., *.py) or dotted path '
-            '(i.e., module.sub_module.factory_function). Verify your input.')
+        if Path(entry_point).suffix == '.yaml':
+            raise Exception('Could not determine the entry point type from value: ' 
```

Comment:
this looks a lot better! :) please change this to ValueError

Reply:
OK